src/store/Simulator.ts

```diff
@@ -0,0 +1,7 @@
+import { defineStore } from "pinia";
+
+export const SimulatorStore = defineStore("main", {
+    state: () => ({
```

Comment:
Sure, I will make a separate folder for the state, action, and getters.

Can we follow the [Vue Style guide](https://v2.vuejs.org/v2/style-guide/?redirect=true#Single-file-component-filename-casing-strongly-recommended) according to which we should use `TitleCase` or `kebab-case`.
We can go with TitleCase as Eslint extensions in text editors supports Title case for auto-completion.
in `.eslintrc.js` - > `eslint:recommended`
also, I have configured Eslint to follow the recommended rules in the above style guide

Reply:
I guess the vue style guide you mentioned above is for vue components only

Reply:
according to this, 
https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/naming-convention.md
It accepts all/several naming conventions.

Reply:
> Lets make this file a folder and inside lets have different files for state, actions & getters, - this way it will be more maintainable when it scales.

Simply splitting the store was causing some errors
Took help from this discussion thread
https://github.com/vuejs/pinia/discussions/1324#discussioncomment-2833429